src/js/pensions/pensions-entry.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createHistory } from 'history';
+import { Router, useRouterHistory } from 'react-router';
+import { Provider } from 'react-redux';
+
+import initReact from '../common/init-react';
+import route from './routes';
+import createCommonStore from '../common/store';
+import createLoginWidget from '../login/login-entry';
+import reducer from './reducer';
+
+require('../common');
```

Comment:
Ah, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) says we can `import '../common` "for side-effects only". Okay.

:man_shrugging: